samza-shell/src/main/bash/run-class.sh

```diff
@@ -97,12 +108,18 @@ else
 fi
 
 if [ -z "$SAMZA_LOG_DIR" ]; then
-  SAMZA_LOG_DIR="$base_dir"
+  # When on samza-yarn, SAMZA_LOG_DIR will point to the symlink located at:
```

Comment:
Thanks for adding the comments. Please remove when on samza-yarn prefix, since this script is applicable only for yarn deployments.

Reply:
Updated to simpify.